util/chunk/chunk.go

```diff
@@ -620,7 +619,8 @@ func (r Row) GetDatum(colIdx int, tp *types.FieldType) types.Datum {
 		}
 	case mysql.TypeNewDecimal:
 		if !r.IsNull(colIdx) {
-			d.SetMysqlDecimal(r.GetMyDecimal(colIdx))
+			dec := *r.GetMyDecimal(colIdx)
```

Comment:
Is there any other place calls Row.GetMyDecimal()?

Reply:
Yes, but only max/min will hold the value directly now.